### DISCUSSION TITLE
Anyone know the function of the consistency VAE ?

### DISCUSSION THREAD

**Discussion by ApolloRay** [PixArt-alpha/PixArt-alpha#120] [Q&A]
In the demo, it offers a consistency VAE, but I'm not sure its function.

**Comment by lawrence-cj** [comment]
It's DALLE3's VAE decode. Put it there for reference.

**Comment by ApolloRay** [comment]
> It's DALLE3's VAE decode. Put it there for reference.

<img width="667" alt="截屏2024-04-17 10 23 58" src="https://github.com/PixArt-alpha/PixArt-alpha/assets/36093263/a779f593-3818-479c-a441-434e814ec9b6">
Can it improve the consistency of the image?
